src/solver/aleatoire/alea_fonctions.h

```diff
@@ -27,9 +27,11 @@
 #ifndef __SOLVER_ALEA_H__
 #define __SOLVER_ALEA_H__
 
+#include "../simulation/sim_structure_donnees.h"
```

Comment:
```
target_include_directories(libantares-solver-aleatoire
		PRIVATE
		${CMAKE_SOURCE_DIR}/solver)
```
Tried this without results

Reply:
Ok, it seems it's because it's included from file src/libs/antares/study/estimate.memory-footprint.cpp:35, which has a (cyclic) dependency on this, and which does not have this include directory.

This will be removed anyway by another PR, let's not worry.